Makefile

```diff
@@ -34,6 +34,9 @@ dev:
 utop: all
 	OCAMLPATH=_build/install/default/lib:$(OCAMLPATH) utop
 
+ocamlformat:
```

Comment:
```suggestion
fmt:
```

Since you deleted `(enabled_for dune)` it's going to format every language Dune supports, i.e. for us it means OCaml and dune files.